modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanel.js

```diff
@@ -147,7 +147,11 @@ const SidebarPanel = React.forwardRef(
 		}));
 
 		return (
-			<Sidebar onClose={onCloseHandle} open={state.isOpen}>
+			<Sidebar
+				fetchSidebarPanelData={() => getData(fetchURL)}
```

Comment:
This is just a small detail but, since we are in a component called sidebar I will name this prop something like 'fetchData'.

Reply:
I like that